ui/src/components/payment/payment-dialog/PaymentDialog.tsx

```diff
@@ -130,15 +139,29 @@ export default class PaymentDialog extends Component<PaymentProps, PaymentState>
     };
 
     private fetchMonthsCountItems = () => {
-        return ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10"];
+        let months = [];
+        for (let i = 1; i < 10; i++) {
+            months.push(i.toString())
+        }
+        return months;
```

Comment:
I'm not sure about it, but I think it's better to directly enter `hardcoded` data instead of calculation of it.

Reply:
yes but hardcode is a bit painfull to see, lets keep it and populate from backend or smth